lib/files.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:path/path.dart';
+const List<String> videoFormats = ['.mp4', '.mov', '.avi', '.wmv'];
```

Comment:
what about all the other extensions? this doesn't cover a lot.

Reply:
this is what we cover for now.

Reply:
.3gp is moved from image to video list :)